src/applications/mhv-medical-records/containers/VaccineDetails.jsx

```diff
@@ -105,7 +105,7 @@ ${record.name}\n
 ${formatNameFirstLast(user.userFullName)}\n
 Date of birth: ${formatUserDob(user)}\n
 ${reportGeneratedBy}\n
-Date received: ${record.date}\n
+Date entered: ${record.date}\n
 ${txtLine}\n\n
 Location: ${record.location}\n`;

```

Comment:
The ticket says that the date should be moved below the `txtLine` to right above the Location field.